src/AppBundle/Factory/CreateObject.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\AppBundle\Factory;
+
+use App\Entity\EducationLevel;
+use App\Entity\User;
+use App\AppBundle\Factory\Creator;
+use Psr\Log\InvalidArgumentException;
+
+class CreateObject implements Creator
+{
+    public function create(string $type)
```

Comment:
poczytaj o Single Resposibility Principle i zastanów się czy ta klasa spełnia tę zasadę.

Reply:
No tak średnio. Poprawię to.